Main.cs

```diff
@@ -66,6 +68,37 @@ private static bool OnToggle(UnityModManager.ModEntry modEntry, bool value)
             return true;
         }
 
+        private static void PatchPersistentJobs()
+        {
+            Type? persistentJobsModInteractionFeaturesType = GetPersistentJobsInteractionType();
+            if (persistentJobsModInteractionFeaturesType != null)
+            {
+                var jobTracksChanged = persistentJobsModInteractionFeaturesType.GetEvent("JobTracksChanged");
+                jobTracksChanged.AddEventHandler(null, new Action<Job>(JobData.JobPatches.UpdateJobsFromPersistentJobs));
+                DebugLog(() => "Persistent Jobs found and hooked");
+            }
+        }
+        /// <summary>
+        /// This will check to see if persistent jobs mod is currently installed and loaded
+        /// </summary>
+        /// <returns>
+        /// Persistent Jobs mod event interface if available, null otherwise.
+        /// </returns>
+        private static Type? GetPersistentJobsInteractionType()
+        {
+            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            {
+                foreach (var type in assembly.GetTypes())
```

Comment:
Any reason to loop through instead of using `assembly.GetType(String)`?

Reply:
I've learned a lot about C# reflection since this code, at the time I didn't know about GetType(String) changed